docs/user/accumulo/ageoff.rst

```diff
@@ -0,0 +1,74 @@
+Age-Off Iterators
+=================
+
+This chapter provides documentation on how to configure the alpha feature for data age-off via Accumulo iterators.
+Currently the only available age off iterator is a date-based age-off iterator which allows administrators to set
+retention periods on data (e.g. 3 months) in order to automatically age-off data from tables without manually deleting
+features.
+
+Installation
+------------
+
+The age-off iterators are provided as part of the GeoMesa Accumulo Distributed Runtime jar which can be found in the
+:doc:`install` chapter.
+
+Configuration
+-------------
+
+Since this feature is currently in alpha, it must be configured via the Accumulo Shell. The following example shows
+how to configure a set of GeoMesa tables with a retention period of 3 months.
+
+.. note::
+
+    Only ``SimpleFeatureType``s with a default date field can be used with the date-based age-off iterator
+
+There are two options that required to configure the iterator:
+
+* **sft** - a GeoMesa ``SimpleFeatureType`` spec string
+* **index** - the GeoMesa Index identifier (usually concatenated index name, colon, and version e.g. z2:2 or records:2)
+* **retention** - an ISO 8601 Durations (or Period) format string
+
+The sft option is a GeoMesa ``SimpleFeatureType`` spec string. You need to scan the catalog table for your to determine
+the correct spec string used by your feature type. We will be using it to set the ``sft`` opt on the age-off iterator.
+It should resemble something like this::
+
+    some_id:String,dtg:Date,geom:Point:srid=4326
```

Comment:
you should include the user data in the string - otherwise the default date might not be calculated correctly. Copying the exact string from the metadata will work - e.g. for example-csv it would be:
`fid:Integer:index=false,name:String:index=true,age:Integer:index=false,lastseen:Date:index=false,*geom:Point:srid=4326;geomesa.index.dtg='lastseen',geomesa.table.sharing='true',geomesa.indices='z3:4:3,z2:3:3,records:2:3,attr:4:3',geomesa.table.sharing.prefix='\\u0001'`